vmdb/config/initializers/as_latch.rb

```diff
@@ -0,0 +1,32 @@
+require 'thread'
+require 'monitor'
+
+if defined?(ActiveSupport::Concurrency)
+  $stderr.puts "Your rails is new enough to have ActiveSupport::Concurrency"
```

Comment:
Not sure when we added it, but it is in 4.2.  So we can delete this when we upgrade (I copied it straight from the release branch)